tests/Support/Traits/InitCommandMockTrait.php

```diff
@@ -9,11 +9,14 @@ trait InitCommandMockTrait
 {
     use MockTrait;
 
-    public function mockFilePutContent(...$arguments): void
-    {
+    public function mockFilePutContent(
```

Comment:
not sure this change required

Reply:
we need to use different envs, with app name in pascal case and without app name in pascal case.

Reply:
@pirs1337 got it!